2024-11-27 00:08:40,474 // WARNING // Неверный тип данных для объекта Runner
Traceback (most recent call last):
  File "C:\Users\Nikita\Desktop\Python - разработчик. Проекты на курсе Urban University\1.1 18.08.2024\pythonProject\test_12_4.py", line 75, in test_run
    blade_runner = Runner(2, 5)
                   ^^^^^^^^^^^^
  File "C:\Users\Nikita\Desktop\Python - разработчик. Проекты на курсе Urban University\1.1 18.08.2024\pythonProject\test_12_4.py", line 15, in __init__
    raise TypeError(f'Имя может быть только строкой, передано {type(name).__name__}')
TypeError: Имя может быть только строкой, передано int
2024-11-27 00:08:40,476 // WARNING // Неверная скорость для Runner
Traceback (most recent call last):
  File "C:\Users\Nikita\Desktop\Python - разработчик. Проекты на курсе Urban University\1.1 18.08.2024\pythonProject\test_12_4.py", line 64, in test_walk
    blade_walker = Runner('Decard', -5)
                   ^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Nikita\Desktop\Python - разработчик. Проекты на курсе Urban University\1.1 18.08.2024\pythonProject\test_12_4.py", line 20, in __init__
    raise ValueError(f'Скорость не может быть отрицательной, сейчас {speed}')
ValueError: Скорость не может быть отрицательной, сейчас -5
